.github/workflows/packaging.yml

```diff
@@ -302,8 +302,14 @@ jobs:
       - name: Extract source archive
         run: |
           tar xf apache-arrow-adbc-${{ steps.info.outputs.VERSION }}.tar.gz
+
+          source ./apache-arrow-adbc-${{ steps.info.outputs.VERSION }}/dev/release/versions.env
+
           mv apache-arrow-adbc-${{ steps.info.outputs.VERSION }} adbc
-          mv apache-arrow-adbc-${{ steps.info.outputs.VERSION }}.tar.gz adbc/ci/linux-packages/
+          # Need to align the file path with the version number (and not the release)
+          cp -r adbc apache-arrow-adbc-${VERSION_NATIVE}
+          tar czf adbc/ci/linux-packages/apache-arrow-adbc-${VERSION_NATIVE}.tar.gz apache-arrow-adbc-${VERSION_NATIVE}
```

Comment:
I can update `ci/linux-packages/` to use `apache-arrow-adbc-${RELEASE}.tar.gz` instead of `apache-arrow-adbc-${VERSION_NATIVE}.tar.gz`.
Should I do this in this branch or in a separated PR?

Reply:
OK! I'll push to this branch!

Reply:
Done.